corehq/ex-submodules/phonelog/tasks.py

```diff
@@ -2,10 +2,13 @@
 from celery.schedules import crontab
 from celery.task import periodic_task
 from django.conf import settings
-from phonelog.models import DeviceReportEntry
+from phonelog.models import DeviceReportEntry, UserErrorEntry, ForceCloseEntry, UserEntry
 
 
 @periodic_task(run_every=crontab(minute=0, hour=0), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
 def purge_old_device_report_entries():
     max_age = datetime.utcnow() - timedelta(days=60)
     DeviceReportEntry.objects.filter(server_date__lt=max_age).delete()
+    UserErrorEntry.objects.filter(server_date__lt=max_age).delete()
+    ForceCloseEntry.objects.filter(server_date__lt=max_age).delete()
+    UserEntry.objects.filter(server_date__lt=max_age).delete()
```

Comment:
I think that makes sense. This made the migration really fast. we can fill those with today's date at a later time.